cognite_toolkit/_cdf_tk/commands/featureflag.py

```diff
@@ -20,6 +20,10 @@ class Flags(Enum):
     INTERNAL: ClassVar[dict[str, Any]] = {"visible": False, "description": "Does nothing"}
     IMPORT_CMD: ClassVar[dict[str, Any]] = {"visible": True, "description": "Enables the import sub application"}
     ASSETS: ClassVar[dict[str, Any]] = {"visible": True, "description": "Enables the support for loading assets"}
+    TIMESERIES: ClassVar[dict[str, Any]] = {
+        "visible": True,
+        "description": "Enables the support for loading timeseries",
+    }
```

Comment:
We already have support for loading timeseries. 
```suggestion
    TIMESERIES_DUMP: ClassVar[dict[str, Any]] = {
        "visible": True,
        "description": "Enables the support for dumping timeseries",
    }
```